src/components/leaderboard/MyStats/LeaderboardProfilePreview.tsx

```diff
@@ -0,0 +1,62 @@
+import AddressAvatar from '@/components/AddressAvatar'
+import Name from '@/components/Name'
+import { mutedTextColorStyles } from '@/components/content-staking/utils/commonStyles'
+import { getProfileQuery } from '@/services/api/query'
+import { cx } from '@/utils/class-names'
+import LeaderboardRoleRadioGroup from '../LeaderboardRadioGroup'
+
+type LeaderboardProfilePreviewProps = {
+  address: string
+  rank: number | null
+}
+
+const LeaderboardProfilePreview = ({
+  address,
+  rank,
+}: LeaderboardProfilePreviewProps) => {
+  const { data: profile } = getProfileQuery.useQuery(address)
+
+  const about = profile?.profileSpace?.content?.about
+
+  return (
+    <div
+      className={cx(
+        'relative flex gap-4 rounded-[20px] border md:flex-col md:items-center',
+        'border-[#6366F1]/20 bg-[#EDF4FF] p-4 dark:bg-[#0053FF]/10'
+      )}
+    >
+      {rank && (
+        <span
+          className={cx(
+            mutedTextColorStyles,
+            'absolute right-4 top-4 font-semibold md:left-4'
+          )}
+        >
+          #{rank + 1}
```

Comment:
better if the function just render rank, without calculation

For examle you want to reuse it without table, so better send already correct number